dali/util/local_file.cc

```diff
@@ -57,6 +61,7 @@ static int get_max_vm_cnt() {
     vm_cnt_sz = std::fread(buffer, 1, MAX_BUFF_SIZE, fp);
     vm_cnt = std::stoi(buffer, nullptr);
   }
+#endif
   return vm_cnt;
```

Comment:
You are returning random number here - initialize vm_cnt to 1

Reply:
I will fix this.

Reply:
fixed